README.md

```diff
@@ -1,5 +1,7 @@
 # Color Vision
 
+![](https://img.shields.io/visual-studio-marketplace/v/wilsonsio.color-vision?style=for-the-badge)
```

Comment:
```suggestion
![Visual Studio Marketplace Rating](https://img.shields.io/visual-studio-marketplace/v/wilsonsio.color-vision)
```

Reply:
Two quick changes, one to add the alttext for the badge and one to remove the style for consistency. Thanks for the contribution.